conda-recipe-openblas/meta.yaml

```diff
@@ -33,7 +33,7 @@ requirements:
    
   run:
     - python {{ PY_VER }}
-    - numpy >=1.16
+    - {{ pin_compatible('numpy') }}
```

Comment:
Looking more closely I see these recipes are not for conda-forge builds, so I don't think the pin_compatible will do anything her. I think that is specific to conda-forge. Maybe you just want to pin to numpy 1.11.* in the build section for these custom conda build scripts.

Reply:
Here is a relevant note in the conda build docs: https://docs.conda.io/projects/conda-build/en/latest/user-guide/tutorials/building-conda-packages.html#edit-the-skeleton-files

Reply:
This looks relevant too: https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html?highlight=numpy